server/Makefile

```diff
@@ -12,7 +12,8 @@ NAMESPACE ?= workspaces-system
 KUBECLI ?= kubectl
 KUSTOMIZE ?= $(BIN_FOLDER)/kustomize
 MOCKGEN ?= $(BIN_FOLDER)/mockgen
-GOLANG_CI ?= go run -modfile $(shell dirname $(ROOT_DIR))/hack/tools/golang-ci/go.mod github.com/golangci/golangci-lint/cmd/golangci-lint
+YQ ?= $(BIN_FOLDER)/yq
+GOLANG_CI ?= $(GO) run -modfile $(shell dirname $(ROOT_DIR))/hack/tools/golang-ci/go.mod github.com/golangci/golangci-lint/cmd/golangci-lint
```

Comment:
as tools are getting more and more it may be valuable introducing a new variable `TOOLS_DIR` set to `$(shell dirname $(ROOT_DIR))/hack/tools`

Reply:
I'll add that in a separate PR; I've got another tool to vendor (`operator-sdk`, needed for the stuff under `operator/`) and I'll make that change in that PR.